14-eco.Rmd

```diff
@@ -610,12 +608,12 @@ In terms of methodology, a few additional points could be addressed:
 - It would be interesting to also model the second ordination axis, and to subsequently find an innovative way of visualizing jointly the modeled scores of the two axes in one prediction map.
 - If we were interested in interpreting the model in an ecologically meaningful way, we should probably use (semi-)parametric models [@muenchow_predictive_2013;@zuur_mixed_2009;@zuur_beginners_2017].
 However, there are at least approaches that help to interpret machine learning models such as random forests (see, e.g., [https://mlr-org.github.io/interpretable-machine-learning-iml-and-mlr/](https://mlr-org.github.io/interpretable-machine-learning-iml-and-mlr/)).
-- A sequential model-based optimization (SMBO) might be preferable to the here used random search for hyperparameter optimization [@probst_hyperparameters_2018]. 
+- A sequential model-based optimization (SMBO) might be preferable to the random search for hyperparameter optimization used in this chapter [@probst_hyperparameters_2018]. 
 
 Finally, please note that random forest and other machine-learning models are frequently used in a setting with lots of observations and many predictors, much more than used in this chapter, and where it is unclear which variables and variable interactions contribute to explaining the response.
 Additionally, the relationships might be highly non-linear.
 In our use case, the relationship between response and predictors are pretty clear, there is only a slight amount of non-linearity and the number of observations and predictors is low.
-Hence, it might be worth to try a linear model.
+Hence, it might be worth it to try a linear model.
```

Comment:
@Robinlovelace Could you pls check this sentence, is it really `it might be worth it`?

Reply:
perfect, thank, Robin!